src/CoreFx.Private.TestUtilities/src/System/Diagnostics/RemoteExecutorTestBase.uap.cs

```diff
@@ -37,7 +40,9 @@ private static RemoteInvokeHandle RemoteInvoke(MethodInfo method, string[] args,
             Type t = method.DeclaringType;
             Assembly a = t.GetTypeInfo().Assembly;
 
-            using (AppServiceConnection remoteExecutionService = new AppServiceConnection())
+            IAsyncOperation<AppServiceResponse> asyncOperation = null;
```

Comment:
> IAsyncOperation<AppServiceResponse> asyncOperation = null; [](start = 11, length = 59)

You should be familiar more than me here, but I am wondering if we just wrap the whole original code of the RemoteInvoke insilde Task.Run and let RemoteInvokeHandle be a wrapper around the returned task. would that be cleaner? or that not work for your scenarios?